core/org.wso2.carbon.utils/src/main/java/org/wso2/carbon/utils/AuditLog.java

```diff
@@ -18,165 +18,156 @@
 
 package org.wso2.carbon.utils;
 
+import org.slf4j.MDC;
+
 import java.time.Instant;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+
+import static org.wso2.carbon.CarbonConstants.LogEventConstants.CORRELATION_ID_MDC;
 
 /**
  * Audit log.
  */
 public class AuditLog {
 
-    private String logId;
-    private Instant recordedAt;
-    private String clientComponent;
-    private String correlationId;
-    private String initiatorId;
-    private String initiatorName;
-    private String initiatorType;
-    private String eventType;
-    private String targetId;
-    private String targetName;
-    private String targetType;
-    private String dataChange;
-
-    public AuditLog(String logId, Instant recordedAt, String clientComponent, String correlationId,
```

Comment:
Removed this without deprecating, since this was merged with https://github.com/wso2/carbon-kernel/pull/3029/files and it is not announced in any public channels, and we didn't advertise it as a feature. Because this audit logs were available only for application mgt component in framework. No one could use this earlier. Because, if we enable this new audit logs, the legacy audit logs will be completely hidden, and only audit logs for app-mgt component will be shown

Reply:
This has been there for about two years now. Even though we haven't announced this to the public, Someone might have already used these (Any community user). So if we remove this, it will break their existing implementation, right? And is it okay to do this kind of API change without deprecating?

Reply:
This is a model class, and we didn't use this new audit log feature. If someone tries to use it by enabling the config, audit logs will be added only for application-mgt operations in framework level. This audit log feature is not yet completed

Reply:
For further clarrifications, added a note in the previous git issue and mentioned the API changes